src/ucp/tag/rndv.c

```diff
@@ -283,7 +394,8 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_get_zcopy, (self),
         length = ucp_mtu - ((uintptr_t)rndv_req->send.buffer % align);
     } else {
         length = ucs_min(rndv_req->send.length - offset,
-                         ucp_ep_config(rndv_req->send.ep)->tag.rndv.max_get_zcopy);
+                         ucs_min(ucp_ep_config(rndv_req->send.ep)->tag.rndv.max_get_zcopy,
+                                 rndv_req->send.length / ucp_ep_config(rndv_req->send.ep)->key.num_rndv_lanes));
```

Comment:
this should be rounded up, to avoid a small remainder. e.g
total_len=9
num_lanes=2
length = ucs_min(..., 9/2=4) = 4
it will be split as 4+4+1

Reply:
aligned